dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.2/com.ibm.websphere.appserver.jaxws-2.2.feature

```diff
@@ -33,7 +33,7 @@ Subsystem-Name: Java Web Services 2.2
  com.ibm.websphere.appserver.classloading-1.0, \
  com.ibm.websphere.appserver.javax.mail-1.5; ibm.tolerates:=1.6, \
  com.ibm.websphere.appserver.globalhandler-1.0, \
- com.ibm.websphere.appserver.jaxb-2.2
+ com.ibm.websphere.appserver.jaxb-2.2; ibm.tolerates:=2.3
```

Comment:
same comment here -- I don't think jaxws-2.2 should be tolerating 2.3.  If we did want to keep the toleration, the tolerates clause will need to be propagated up to all features that include jaxws-2.2 (since toleration of public features is not transitive)